app/assets/src/components/views/discovery/InfoBanner.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import { logAnalyticsEvent } from "~/api/analytics";
+import Link from "~ui/controls/Link";
+import NoResultsIcon from "~ui/icons/NoResultsIcon";
+
+import cs from "./info_banner.scss";
+
+const InfoBanner = ({
+  className,
+  icon,
+  iconClassName,
+  link,
+  message,
+  messageClassName,
+  suggestion,
+  title,
+  titleClassName,
+  type,
+}) => {
+  // This is a hack to associate the event with the parent component, DiscoveryView
```

Comment:
👍 also it's hack because we are logging a non-interaction event

Reply:
It looks like this event log call was copied from different components a copy times now... I'm glad you added the type ... an alternative would be to rename the event because it has lost its original meaning.

Reply:
We can log them all on the clients... probably the best since it is now generic.

As you say, this is also a different type of a event. It is an impression vs. an interaction. We should have that distinction as part of the event at some point.